pkg/storage/stores/shipper/uploads/table.go

```diff
@@ -35,7 +36,7 @@ const (
 	snapshotFileSuffix = ".snapshot"
 )
 
-var bucketName = []byte("index")
+var defaultBucketName = []byte("index")
```

Comment:
How do you feel about calling this the IndexBucketName and making it public? We have it in both `local` and `uploads` package, can it be shared or are they referencing different things?

```suggestion
var IndexBucketName = []byte("index")
```

Reply:
yeah, they are the same.